### ISSUE TITLE
Pushing image to container registry

### ISSUE THREAD

**Issue by ryantwarner** [MicrosoftDocs/mixed-reality#534]
Is this step necessary and if it is has it been missed?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c504714b-acd8-e790-201f-c987cd47d78f
* Version Independent ID: 794e8493-5b18-9a94-bfbc-2d030ed37ae8
* Content: [MR and Azure 313 - IoT Hub Service - Mixed Reality](https://docs.microsoft.com/en-us/windows/mixed-reality/mr-azure-313#chapter-2---the-container-registry-service)
* Content Source: [mixed-reality-docs/mr-azure-313.md](https://github.com/MicrosoftDocs/mixed-reality/blob/master/mixed-reality-docs/mr-azure-313.md)
* Product: **windows**
* GitHub Login: @DrNeil
* Microsoft Alias: **v-arood**

**Comment by StefanoDeflorio** [comment]
Im not sure I understand the question, chapter 8 and 9 instructions are to follow to push the container to Container Registry and instruct the IoT Edge Service to use the image stored in the CR for deployment.

**Comment by ryantwarner** [comment]
Hi Stefano, sorry for being unclear, and thank you for the reply. I followed the directions in chapters 7, 8 and 9; I was only seeing edgeHub and edgeAgent on my linux host. My container registry was showing 0 bytes used, until I manually pushed the image in powershell. Once I had done that and deployed again I saw my device show up and was able to complete the tutorial. I'm not sure if I missed a step or if the step was taken care of automatically and I was unlucky. Thanks again!
